Add konsole colorscheme

muditjoshi98 opened the pull request:
Added colorscheme for konsole also using help from iterm2 theme

**Darker**
![Darker](https://user-images.githubusercontent.com/33528239/80596353-df2e0b00-8a43-11ea-8760-03a07130dff3.png)
**Default**
![Default](https://user-images.githubusercontent.com/33528239/80596361-e228fb80-8a43-11ea-957d-345a634d8541.png)
**Lighter**
![Lighter](https://user-images.githubusercontent.com/33528239/80596368-e35a2880-8a43-11ea-974e-b95e38c19043.png)
**Ocean**
![Ocean](https://user-images.githubusercontent.com/33528239/80596375-e523ec00-8a43-11ea-9d99-f400718f1983.png)
**Palenight**
![Palenight](https://user-images.githubusercontent.com/33528239/80596386-e6551900-8a43-11ea-9cfe-733b810056b3.png)

muditjoshi98 committed:
Add konsole colorscheme

kaicataldo commented:
Thanks for the PR! Mind sharing some screenshots since I don't have a machine running KDE? This will also help for posterity.

muditjoshi98 committed:
Changes in light colors and added screenshots

muditjoshi98 commented:
Hi @kaicataldo,
I have added screenshots in README.md  for posterity and also done few changes in colors using help from [here](https://github.com/material-theme/vsc-material-theme/tree/master/scripts/generator/settings/specific).
I have also added screenshots in PR comment.

kaicataldo reviewed (approved):
LGTM, thank you very much!
